docker-compose.yml

```diff
@@ -4,6 +4,13 @@
 
 version: '2'
 services:
+  amazoneventbridge:
+    image: localstack/localstack:latest
```

Comment:
Interesting.  Maybe we should consider using this for integration testing with the other AWS service connectors too.

https://hub.docker.com/r/localstack/localstack/

/cc @ennru

Reply:
I had to actually wait a while for raising this PR, as the support of eventBridge in localstack was broken. I would prefer pinning it into specific version but at the time of raising the PR the previous release was not working correctly only the `latest` was.